helix-term/src/commands/typed.rs

```diff
@@ -2304,37 +2304,37 @@ fn reset_diff_change(
 
     let diff = handle.load();
     let doc_text = doc.text().slice(..);
-    let line = doc.selection(view.id).primary().cursor_line(doc_text);
-
-    let Some(hunk_idx) = diff.hunk_at(line as u32, true) else {
-        bail!("There is no change at the cursor")
-    };
-    let hunk = diff.nth_hunk(hunk_idx);
+    let (start_line, end_line) = doc.selection(view.id).primary().line_range(doc_text);
```

Comment:
Could we make this multicursor aware, I could imagine that being useful.

I think commands that are selection instead of cursor based should respect multi cursors